pkg/archive/archive_linux.go

```diff
@@ -2,22 +2,29 @@ package archive // import "github.com/docker/docker/pkg/archive"
 
 import (
 	"archive/tar"
+	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
+	"syscall"
 
+	"github.com/containerd/continuity/fs"
 	"github.com/docker/docker/pkg/system"
+	"github.com/pkg/errors"
 	"golang.org/x/sys/unix"
 )
 
-func getWhiteoutConverter(format WhiteoutFormat) tarWhiteoutConverter {
+func getWhiteoutConverter(format WhiteoutFormat, inUserNS bool) tarWhiteoutConverter {
```

Comment:
(only glancing over the code); It's not an exported function, so not a big deal; just thinking if it would be cleaner to pass the `options` struct, instead of adding a boolean

Reply:
What would be other fields of `options`?

Reply:
not sure  😅 . My train of thought was that in case more options appeared in future, we didn't have to change the signature